config.yaml

```diff
@@ -303,6 +303,10 @@ options:
     default: false
     type: boolean
     description: Enable timescaledb extension
+  plugin_audit_enable:
+    default: false
```

Comment:
JFYI, Audit plugin has been enabled on MySQL by default (for login actions only). Please check with @7annaba3l